app/src/main/res/values/strings.xml

```diff
@@ -320,6 +320,7 @@
     <string name="two_factors_otp_error">Two factors OTP is required</string>
     <string name="no_issue">No issue</string>
     <string name="copy">Copy</string>
+    <string name="copy_repo_url">Copy repo URL</string>
```

Comment:
Yeah, take out copy repo url and use `<string name="copy">Copy</string>`
you can change it to `<string name="copy">Copy Url</string>`